waiter/src/waiter/util/utils.clj

```diff
@@ -45,6 +45,14 @@
            (org.joda.time DateTime)
            (schema.utils ValidationError)))
 
+;; Envoy "%RESPONSE_FLAGS%" values returned by Raven for error diagnostics
+;; https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#config-access-log-format-response-flags
+(def ^:const envoy-empty-response-flags "-")
+(def ^:const envoy-stream-idle-timeout "SI")
+(def ^:const envoy-upstream-connection-failure "UF")
+(def ^:const envoy-upstream-connection-termination "UC")
+(def ^:const envoy-upstream-request-timeout "UT")
```

Comment:
`status_codes.clj` feels a better location for these constants

Reply:
Moved